src/component/index.js

```diff
@@ -53,10 +53,10 @@ export default class Toggle extends Component {
         onTouchEnd={this.handleClick}>
         <div className='react-toggle-track'>
           <div className='react-toggle-track-check'>
-            <Check />
+            {this.props.icons && this.props.icons.checked}
           </div>
           <div className='react-toggle-track-x'>
-            <X />
+            {this.props.icons && this.props.icons.unchecked}
```

Comment:
Could you rebase and use the destructuring above (`const { className, ...inputProps } = this.props`) to make this cleaner and prevent `icons` from being passed down to the `input`-element? (That wasn't there when you wrote this, so you didn't miss it ;))